android/src/main/java/com/rnbiometrics/ReactNativeBiometrics.java

```diff
@@ -147,6 +182,99 @@ public void deleteKeys(Promise promise) {
         }
     }
 
+    @ReactMethod
+    public void deleteEncryptionKey(Promise promise) {
+        boolean deletionSuccessful = false;
+        if (doesBiometricEncryptionKeyExist()) {
+            deletionSuccessful = deleteBiometricEncryptionKey();
+            if (!deletionSuccessful) {
+                promise.reject("Error deleting biometric encryption key", "Error deleting biometric encryption key");
+            }
+        }
+        WritableMap resultMap = new WritableNativeMap();
+        resultMap.putBoolean("keysDeleted", deletionSuccessful);
+        promise.resolve(resultMap);
+    }
+
+    @ReactMethod
+    public void encryptData(final ReadableMap params, final Promise promise) {
+        UiThreadUtil.runOnUiThread(
+                new Runnable() {
+                    @Override
+                    public void run() {
+                        try {
+                            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
+                                throw new Exception("Cannot generate keys on android versions below 6.0");
```

Comment:
It's more or less a dummy message to make linters happy; since no keys can be generated in the first place, this could _should_ be unreachable. I fixed the text anyway.